### PR TITLE
Option to enable/disable layer duration chart scrolling & some trailing whitespaces my editor removed

### PR DESCRIPTION
**Pull request by Willmac16**

### CODE REVIEW

File: octoprint_dashboard/static/js/dashboard.js [hunk 1]
```diff
@@ -637,16 +643,22 @@ $(function () {
                     showGrid: false,
                     showLabel: true,
                     labelInterpolationFnc: function skipLabels(value, index, labels) {
-                        let interval = 5;
-                         if (index % interval == 0) {
-                             return value;
-                         } else {
-                             return null;
+                        var interval;
```
Comment (JeffB42) [inline octoprint_dashboard/static/js/dashboard.js:646]: let and const (let in this case) is usually preferred over var (no hoisting)

File: octoprint_dashboard/static/js/dashboard.js [hunk 2]
```diff
@@ -637,16 +643,22 @@ $(function () {
...
+                        if (self.settingsViewModel.settings.plugins.dashboard.enableLayerGraphScroll()) {
+                            interval = 5;
+                        } else {
+                            interval = Math.ceil(labels.length/20);
```
Comment (JeffB42) [inline octoprint_dashboard/static/js/dashboard.js:650]: You could use a ternary operator here:
```
const interval = (self.settingsViewModel.settings.plugins.dashboard.enableLayerGraphScroll())
  ? 5
  : Math.ceil(labels.length/20);

File: octoprint_dashboard/static/js/dashboard.js [hunk 3]
```diff
@@ -637,16 +643,22 @@ $(function () {
...
+                        }
+
+                        if (value % interval == 0) {
```
Comment (JeffB42) [inline octoprint_dashboard/static/js/dashboard.js:653]: In JavaScript, use === and !== over == and !=

**Comment by StefanCohen** [comment]
Thanks for the PR! I will unfortunately not have time to review for a while as I'm in the middle of a construction projet but will let you know once I'm back.

**Comment by j7126** [comment]
I think you will need to add that checkbox field into the backend init py before merging so that the setting actually gets saved.

**Pull request closed without merging**